packages/local-environment/package.json

```diff
@@ -16,6 +16,7 @@
     "agents_stop": "docker ps --filter name=_agent -aq | xargs docker rm --force",
     "agents_stop_running": "docker ps --filter name=_agent --filter status=running -aq | xargs docker stop",
     "build": "tsc --build",
+    "bootup": "chmod ug+x ./bootup.sh && ./bootup.sh",
```

Comment:
@Imti I'm pretty sure you can commit files with permissioning applied so if you do `chmod ug+x ./bootup.sh` and commit the file you won't need to do this every time.